### PR TITLE
Reduce redundant code in procedure declarations

### PR DESCRIPTION
**Pull request by findepi**

### CODE REVIEW

File: plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/procedure/FlushMetadataCacheProcedure.java [hunk 1]
```diff
@@ -69,17 +69,8 @@ public class FlushMetadataCacheProcedure
...
-    static {
-        try {
-            FLUSH_HIVE_METASTORE_CACHE = lookup().unreflect(FlushMetadataCacheProcedure.class.getMethod(
-                    "flushMetadataCache", String.class, String.class, List.class, List.class, List.class, List.class));
-        }
-        catch (ReflectiveOperationException e) {
-            throw new AssertionError(e);
-        }
-    }
+    private static final MethodHandle FLUSH_HIVE_METASTORE_CACHE = Reflection.methodHandle(FlushMetadataCacheProcedure.class,
```
Comment (hashhar) [inline plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/procedure/FlushMetadataCacheProcedure.java:72]: nit: static import

**Review by hashhar — COMMENTED** [review]
I'll re-read the 2nd last commit, rest looks good.

**Comment by findepi** [comment]
pushed import change (https://github.com/trinodb/trino/pull/18569#discussion_r1286792992)
build was green (https://github.com/trinodb/trino/actions/runs/5786264959/job/15680806684?pr=18569)

**Review by hashhar — APPROVED** [review]
LGTM

**Pull request merged**